microcosm_pubsub/codecs.py

```diff
@@ -39,39 +40,17 @@ class PubSubMessageSchema(Schema):
     Base schema for messages, including a media type.
 
     """
-    media_type = fields.Method(
-        # called by dump
-        serialize="serialize_media_type",
-        # called by load
-        deserialize="deserialize_media_type",
-        attribute="mediaType",
-        # need to set missing to non-None or marshmallow won't call the deserialize function
-        missing=DEFAULT_MEDIA_TYPE,
+    MEDIA_TYPE = DEFAULT_MEDIA_TYPE
```

Comment:
Is `MEDIA_TYPE` an attribute of the base class that we're overriding here?

Reply:
It should be overridden on all subclasses, yeah. Either that, or it can be made an abstract property of the base class, and overridden for all subclasses.